### DISCUSSION TITLE
Detection against self responses

### DISCUSSION THREAD

**Discussion by freyresoft** [oobabooga/text-generation-webui#818] [General]
Hello,

Is there any option to avoid model self responses? I have tried alpaca-30b-lora-int4 , vicuna-13b-GPTQ-4bit-128g and a few others and most of the times after the correct answer I get extra text that have nothing to do with the question.

Thanks

**Comment by antthor** [comment]
I think this was mentioned in an issue here: https://github.com/oobabooga/text-generation-webui/issues/805#issuecomment-1498117630
But I've been unable to get any fixes working for it.

**Comment by mcmonkey4eva** [comment]
See my comment here: https://github.com/oobabooga/text-generation-webui/discussions/839#discussioncomment-5557486 wherein I explain a bit of why some models do that, and what the options are to avoid it.
